### PR TITLE
[DC-3631] Update cope_survey to reduce false positive results due to concept domains

### PR DESCRIPTION
**Pull request by MichaelSchmidt82**
**Background:**
"The issue with the `procedure_occurence` covid vaccine concepts failing due to 'being suppressed in RT' is caused by the vocabulary `domain_id`'s for the associated Standard concepts which populate `procedure_concept_id` (failing column) being 'Drug' in the combined stage. These records are not actually suppressed they get moved to the Drug table by a combined CR."

**This PR:**
Updated to look for concepts that are being suppressed in the deid stage. 
- If they existed at combined stage, they should still be in deid.
- If they did not exist in combined stage, they should not be expected to exist in died.

See Jira issue [DC-3631](https://precisionmedicineinitiative.atlassian.net/browse/DC-3631) for the complete description and scope
See Jira issue [DC-3625](https://precisionmedicineinitiative.atlassian.net/browse/DC-3625) for the investigation.

[DC-3631]: https://precisionmedicineinitiative.atlassian.net/browse/DC-3631?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DC-3625]: https://precisionmedicineinitiative.atlassian.net/browse/DC-3625?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Helpful:**  Changes occur around lines 312 / 309 lower

### CODE REVIEW

File: data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py [hunk 1]
```diff
@@ -290,127 +286,137 @@
...
-FROM `{{project_id}}.{{deid_cdr}}.{{table_name}}` c
-JOIN  `{{project_id}}.{{deid_cdr}}.concept` on concept_id={{column_name}}
- WHERE (
-        -- done by name and vocab -- -- this alone should be enough, no need for others --
-        REGEXP_CONTAINS(concept_name, r'(?i)(COVID)') AND
-        REGEXP_CONTAINS(concept_name, r'(?i)(VAC)') AND 
-        vocabulary_id not in ('PPI')
-    ) OR (
-        -- done by code  and vocab --
+FROM `{{project_id}}.{{deid_cdr}}.procedure_occurrence` c
```
Comment (ksdkalluri) [inline data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py:356]: Here it should not be `procedure_occurrence` it should be `{{table_name}}`. The table_name in the ticket is added for using it as example and testing. if you are looping through tables and columns for in your case this should be templated to as all other ones.

File: data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py [hunk 2]
```diff
@@ -290,127 +286,137 @@
...
+JOIN (
+  SELECT concept_id as concept_id_in_combined
+        FROM `{{project_id}}.{{com_cdr}}.procedure_occurrence` c
```
Comment (ksdkalluri) [inline data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py:359]: Same here with the `procedure_occurrence`, it should be `{{table_name}}`.

File: data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py [hunk 3]
```diff
@@ -29,11 +29,12 @@
...
-project_id = ""
-com_cdr = ""
-deid_cdr = ""
-sandbox=""
-run_as=""
+project_id = ""  # The project to examine
+com_cdr = ""  # The comibend dataset
+deid_cdr = ""  # the deid dataset
+sandbox = "" # sandbox dataset
+pipeline = "" # the pipeline tables
```
Comment (ksdkalluri) [inline data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py:36]: Why was this added to the notebook if it is used no where in the code?

**Review by ksdkalluri — CHANGES_REQUESTED** [review]
@MichaelSchmidt82 I requested for few changes, once you make the changes please test the updates and push them.

**Review by ksdkalluri — CHANGES_REQUESTED** [review]

File: data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py [hunk 4]
```diff
@@ -29,11 +29,11 @@
...
 # Parameters
-project_id = ""
-com_cdr = ""
-deid_cdr = ""
-sandbox=""
-run_as=""
+project_id = ""  # The project to examine
+com_cdr = ""  # The comibend dataset
+deid_cdr = ""  # the deid dataset
+sandbox = "" # sandbox dataset
```
[the remark this answers is not in the document]
Comment (MichaelSchmidt82) [reply data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py:35]: Based on your comment earlier, I put this back.  The description should be better, so I'll clarify.

**Review by ksdkalluri — APPROVED** [review]

**Pull request merged**